Source/Private/ASDisplayNode+UIViewBridge.mm

```diff
@@ -913,6 +945,22 @@ - (void)setAccessibilityHint:(NSString *)accessibilityHint
 {
   _bridge_prologue_write;
   _setAccessibilityToViewAndProperty(_accessibilityHint, accessibilityHint, accessibilityHint, accessibilityHint);
+  if (AS_AT_LEAST_IOS11) {
+    _setAttributedAccessibilityToViewAndProperty(_accessibilityAttributedHint, [[NSAttributedString alloc] initWithString:accessibilityHint], @"accessibilityAttributedHint", [[NSAttributedString alloc] initWithString:accessibilityHint]);
```

Comment:
Nit: For readability and a tiny perf win, cache `[[NSAttributedString alloc] initWithString:accessibilityHint]` instead of allocating 2 identical objects.